freebasics/templates/search/search_results.html

```diff
@@ -5,30 +5,21 @@
 {% if search_results %}
 <div class="block-search fb-body fb-card">
   <h6 class="block-heading fb-accent-1">Search Results</h6>
```

Comment:
Can you just double check that strings in all these templates are in trans blocks